tests/test_lambda_runtime_marshaller.py

```diff
@@ -38,10 +60,26 @@ def test_json_serializer_is_not_default_json(self):
         self.assertFalse(hasattr(stock_json, "YOLO"))
         self.assertTrue(hasattr(simplejson, "YOLO"))
 
-    def test_to_json_unicode_encoding(self):
+    @parameterized.expand(
+        execution_envs_is_enabled_lambda_marshaller_ensure_ascii_false
+    )
+    def test_to_json_unicode_not_escaped_encoding(self, execution_env):
+        os.environ = {"AWS_EXECUTION_ENV": execution_env}
         response = to_json({"price": "£1.00"})
         self.assertEqual('{"price": "£1.00"}', response)
         self.assertNotEqual('{"price": "\\u00a31.00"}', response)
         self.assertEqual(
             19, len(response.encode("utf-8"))
         )  # would be 23 bytes if a unicode escape was returned
+
+    @parameterized.expand(
+        execution_envs_not_enabled_lambda_marshaller_ensure_ascii_false
```

Comment:
nit: `execution_envs_not_enabled_lambda_marshaller_ensure_ascii_true` variable naming makes it more clear that ensure_ascii=True if not explict set to false.